pandas/tests/frame/test_misc_api.py

```diff
@@ -61,37 +61,32 @@ def test_join_index(self):
         # left / right
 
         f = self.frame.reindex(columns=['A', 'B'])[:10]
-        f2 = self.frame.reindex(columns=['C', 'D'])
+        f2 = self.frame.reindex(columns=['C', 'D'])[5:][::-1]
```

Comment:
If .ix is deprecated, this should be pointed out in the doc:
- http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.ix.html
- http://pandas.pydata.org/pandas-docs/stable/indexing.html